### PR TITLE
Bug: Fix taskbar hover icon

### LINKED ISSUE #9925: Bug: App has wrong icon when hovering over taskbar icon
### Requirements
- use appWindow.SetIcon(path_to_ico)

### PR DESCRIPTION
**Pull request by blouflashdb**
**Resolved / Related Issues**
Items resolved / related issues by this PR.
- Closes #9925 

**Validation**
How did you test these changes?
- [x] Built and ran the app
- [ ] Tested the changes for accessibility

**Screenshots (optional)**
[Add screenshots here.](https://i.imgur.com/qfPG25e.png)

### CODE REVIEW

File: src/Files.App/Helpers/FilePropertiesHelpers.cs [hunk 1]
```diff
@@ -68,6 +69,9 @@ public static async Task OpenPropertiesWindowAsync(object item, IShellPage assoc
                     var propertiesWindow = new WinUIEx.WindowEx();
                     var appWindow = propertiesWindow.AppWindow;
 
+                    // Set icon
+                    appWindow.SetIcon(Path.Combine(Package.Current.InstalledLocation.Path, "Assets/AppTiles/StoreLogo.ico"));
```
Comment (yaichenbaum) [inline src/Files.App/Helpers/FilePropertiesHelpers.cs:73]: We'd like to use a method that works with png but happy to fall back on this if there isn't any other choice.

We should however have an icon for each version of the app (dev, preview and release)

**Comment by blouflashdb** [comment]
Will add Icons for preview dev & release

File: src/Files.App/MainWindow.xaml.cs [hunk 2]
```diff
@@ -46,6 +48,9 @@ private void EnsureEarlyWindow()
             // Set title
             AppWindow.Title = "Files";
 
+            // Set icon
+            AppWindow.SetIcon(Path.Combine(Package.Current.InstalledLocation.Path, "Assets/AppTiles/StoreLogo.ico"));
```
Comment (yaichenbaum) [inline src/Files.App/MainWindow.xaml.cs:52]:
```suggestion
            AppWindow.SetIcon(Path.Combine(Package.Current.InstalledLocation.Path, "Assets/AppTiles/Dev/Logo.ico"));
```
There should be a similar logo in the `Release` and `Preview` folder.

Comment (yaichenbaum) [reply src/Files.App/MainWindow.xaml.cs:52]: The build pipeline will automatically change the path in the code when it builds the release/preview version of Files.

**Review by yaichenbaum — APPROVED** [review]
LGTM

**Pull request merged**